common.sh

```diff
@@ -144,7 +143,8 @@ function build_kb {
   GO111MODULE=on go build $opts -o $tmp_root/kubebuilder/bin/kubebuilder ./cmd
 }
 
-function fetch_go_tools {
+# Required to install the go dep in the https://prow.k8s.io/.
+function install_dep_by_git {
```

Comment:
Why we use 2 different approaches to install dep?

Reply:
Because after the upgrade to use go version 1.13 these steps are not working after for the CI tests and prow will face the GOBIN issue.

Reply:
PS.: Note that it shows that will be removed in the future versions as well, when we no longer support dep.

Reply:
Can you please describe the GOBIN issue with prow?
I'd suggest move put some description in the comments.